docs/developmentSetup.md

```diff
@@ -68,14 +70,16 @@ To submit challenges:
 3. Submit your code `c0d3 submit --url http://localhost:4000/api/graphql`
 It should be visible on your local setup.
 
-To open terminal inside container:
-1. Check your current active docker containers `docker ps`. There should be four of them - web, db and app, postgres.
-2. Open bash console inside postgres container `docker exec -it * postgres id* /bin/bash`.
-3. You can connect to postgres `psql -U *username* -d *dbname*`. If you have psql installed you can do it the easier way `psql -U c0d3_admin -h localhost -p 7000 -d c0d3`
+To connect to postgres database inside container:
+1. Check your current active docker containers `docker ps`. Notice docker_postgres container id (first few characters are enough).
+2. Open bash console inside postgres container `docker exec -it *postgres id* /bin/bash`.
+3. Connect to postgres `psql -U c0d3_admin -d c0d3`. 
+
+If you have psql installed locally you can type `psql -U c0d3_admin -h localhost -p 7000 -d c0d3` with the same result.
 
-If you want to pass further lessons as admin you need to manually edit database (since no one can review your submissions). Connect to postgres container (see above) and type:
+If you want to pass further lessons as an admin you need to manually edit database (since no one can review your submissions). Connect to postgres container and type:
```

Comment:
I think it would be better to have an user with all lessons completed already, so we don't need to go over these steps. Could be the `admin` you already have in the dump.

Maybe include a normal user account as well.

Reply:
There are two premade users now. `admin:password` which can review any submission and `newbie:password` which has just registered.